test/api/cpp/CMakeLists.txt

```diff
@@ -36,86 +34,29 @@ add_custom_target(apitests
 
 set(CVC5_API_TEST_FLAGS -D__BUILDING_CVC5_API_TEST -Dcvc5_obj_EXPORTS)
 
-macro(cvc5_add_api_test name)
-  set(test_bin_dir ${CMAKE_BINARY_DIR}/bin/test/api/cpp)
+macro(cvc5_add_api_test name dir)
+  set(test_bin_dir ${CMAKE_BINARY_DIR}/bin/test/api/cpp/${dir})
   add_executable(${name} ${name}.cpp)
   target_link_libraries(${name} PUBLIC main-test)
   target_compile_definitions(${name} PRIVATE ${CVC5_API_TEST_FLAGS})
   set_target_properties(${name}
     PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${test_bin_dir})
-  add_test(api/cpp/${name} ${ENV_PATH_CMD} ${test_bin_dir}/${name})
-  set_tests_properties(api/cpp/${name} PROPERTIES LABELS "api")
+  if("${dir}" STREQUAL "")
+    set(test_name api/cpp/${name})
+  else()
+    set(test_name api/cpp/${dir}/${name})
+  endif()
```

Comment:
For consistency, you should add a similar conditional statement in the `test/api/c/CMakeLists.txt` file, where the filename is `capi_${name}`.